static/js/ui-utils.js

```diff
@@ -7,6 +7,7 @@ function rawBGsEnabled() {
 }
 
 function getBrowserSettings(storage) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['getBrowserSettings' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/800382164/issues/source?bid=2452379&fileBranchId=2725437#l9)